### ISSUE TITLE
Deleting my code when using the shortcut to delete the console

### ISSUE THREAD

**Issue by lainbo** [Chakroun-Anas/turbo-console-log#219]
```html
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>

<body>
  <script>
    const { open, onChange } = useFileDialog({
      accept: '.szjc',
    })
    console.log("open:", open)

  </script>
</body>

</html>

```

When using the shortcut key to delete the log message, the entire `useFileDialog` will be deleted. In this example, after pressing the shortcut key, only an empty script tag will remain.

**Comment by KnifeFed** [comment]
Yeah, it deletes every line containing the letters "log" in succession. Kind of a big oversight.

**Comment by Chakroun-Anas** [comment]
Hi @lainbo @KnifeFed, 

I’ve tested this behavior in the latest version of Turbo Console Log and couldn’t reproduce the issue. It seems the problem may no longer exist.

For additional context, the current detection logic in the extension ensures that only log messages matching the defined logFunction, logMessagePrefix, or delimiterInsideMessage patterns are identified and processed. The function strictly applies the following steps:

- Escapes special characters in the logFunction to avoid accidental matches.
- Verifies each line in the document against the expected logFunction using a regular expression.
- Checks if the log message contains either the logMessagePrefix or delimiterInsideMessage before performing any operations.

This stricter approach ensures that only intended log messages are matched, preventing accidental deletions of unrelated lines like those containing parts of functions such as useFileDialog.

Could you please verify this from your side with the latest version? If the issue persists, sharing additional details (e.g., your VS Code version, specific steps to reproduce, or any custom settings) would be greatly helpful.

Thanks for raising this, and I look forward to hearing back from you!
